inc/editor/blocks/toc.php

```diff
@@ -34,7 +34,9 @@ function get_headings_from_post_content( string $content ) : array {
 	// within PHP rendering code, even once the content is parsed as blocks.
 	// DOMDocument is the most reliable tool to locate the values we want.
 	$heading_block_doc = new DOMDocument();
+	libxml_use_internal_errors( true ); // Silence warnings.
```

Comment:
```suggestion
	// DOMDocument raises warnings when encountering HTML5 elements.
	// Silence those warnings because unknown tags are not an issue.
	libxml_use_internal_errors( true );
```
What do you think about adding a bit more documentation?